gensim/matutils.py

```diff
@@ -382,6 +383,110 @@ def cossim(vec1, vec2):
     return result
 
 
+def isbow(vec):
+    """
+    Checks if vector passed is in bag of words representation or not.
+    """
+    if scipy.sparse.issparse(vec):
```

Comment:
I only check if it's in sparse format because I want to convert it to a list - because it's easier working with lists - especially because I was iterating over lists in the previous implementation. 
If it is actually sparse or not shouldn't make a difference to checking if it's a bow - right?